### DISCUSSION TITLE
support for .mts files ?

### DISCUSSION THREAD

**Discussion by ksXV** [nvim-treesitter/nvim-treesitter#3545] [Q&A] [answered]
### Describe the highlighting problem

So I am writing a node server in ts and I need to use .mts instead of .ts thing is I see that treesitter doesn t have support for .mts files ? 
Is there any bypass to this or ?

### Example snippet that causes the problem

any code written in a .mts file

### Tree-sitter parsing result

everything is gray

### Example screenshot

![image](https://user-images.githubusercontent.com/90782294/192160267-7202e90a-c067-4d7c-9518-fe801087aa6e.png)

### Output of `:checkhealth nvim-treesitter`

```markdown
https://cdn.discordapp.com/attachments/494592039024066560/1023668615012175992/unknown.png
```

### Output of `nvim --version`

```text
https://media.discordapp.net/attachments/494592039024066560/1023668855505158254/unknown.png?width=1025&height=301
```

**Comment by ObserverOfTime** [comment, marked as answer]
Fixed in neovim/neovim@fe0727a

**Reply by theHamsta** [reply]
You can just set the file type to typescript manually when your nvim version does not have above mentioned fix
